packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -487,6 +477,21 @@ Router = Router.extend({
     return false;
   },
 
+  createReactViewHandler(routeName, Path, additionalParams) {
```

Comment:
Is `Path` used? I can't see it.
I do see one instance of `this.createReactViewHandler('cookies_disabled', {…})` above on L176 that seems to be passing some additional params (apart from `routeName`) so I was wondering if those are just getting dropped since they might be getting passed as `Path` and not `additionalParams` argument.

Reply:
Oh that's a good catch -- yeah I was trying to mirror the pre-existing method but I think you're right, this would be an improvement. I will update!